data/scrape/lib/generate_summary.py

```diff
@@ -5,10 +5,10 @@
 import logging
 import operator
 import os
-
+import pandas
```

Comment:
Imports are out of order. Pandas is a 3rd-party module and doesn't belong mixed in with the standard libraries. [See here](https://peps.python.org/pep-0008/#imports).

Run `isort` on this file or fix it manually.